source/slang/slang-emit-cpp.cpp

```diff
@@ -2326,6 +2356,14 @@ bool CPPSourceEmitter::tryEmitInstExprImpl(IRInst* inst, const EmitOpInfo& inOut
             m_writer->emit("))");
             return true;
         }
+        case kIROp_StringLit:
+        {
+            m_writer->emit("String(");
+            m_writer->emit(Slang::Misc::EscapeStringLiteral(as<IRStringLit>(inst)->getStringSlice()));
```

Comment:
As above would probly be better to use SlangEscapeUtil.

Reply:
Yeah, I did notice this class but I am not sure how to use it.